json-lint/utils/schemaUtils.js

```diff
@@ -25,7 +25,7 @@ const getErrorMessage = (command, jsonPath, status, execPath) => `Error merging
 function getMarfeelExtendedJson(jsonPath, command = JSON_MERGE_COMMAND ) {
 	const mrfJson = spawnSync(command , [ jsonPath ]);
 	
-	if(mrfJson.stderr.length > 0 ||!!mrfJson.status){
+	if((mrfJson.stderr && mrfJson.stderr.length > 0) ||!!mrfJson.status){
```

Comment:
const (and clean code) are for free :P
```suggestion
        const hasErrors = mrfJson.stderr && mrfJson.stderr.length > 0;
	if(hasErrors || mrfJson.status !== 0) {
```